### ISSUE TITLE
Modal Component Accessibility issue

### ISSUE THREAD

**Issue by AGarcia-Clarity** [department-of-veterans-affairs/vets-design-system-documentation#571]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened
When using the WAVE browser extension I noticed there is an unintended accessibility issue within the Modal component.  The lack of the `status` property causes the close button's `aria-label` to read as **Close the undefined modal**.  A custom `status` property item causes the Modal to break.

![Screen Shot 2021-11-04 at 12 20 48 PM](https://user-images.githubusercontent.com/67602137/140378484-c54db2bd-7691-4e07-8dee-730fdd3684af.png)

## What I expected to happen
- I would expect a default for a Modal without a `status` property (example: **Close the modal**)
OR
- I would expect the ability to utilize a custom `status` property (example: **Close the VA.gov Sign in modal**) 

## Reproducing
- Formation version: latest in vets-website (`^3.3.4`
- Device: Macbook (any device will be affected
- Browser: Chrome (any browser will be affected)

Steps to reproduce:

1. Navigate to [staging.va.gov](https://staging.va.gov)
2. Click on the **Sign in** link to open the Sign In Modal
3. Open up the DevTools and look at the `aria-label` attribute and it should say `Close the undefined modal`

**Comment by bkjohnson** [comment]
I'm having a hard time reproducing this. The `aria-label` I see just says "close".

![image](https://user-images.githubusercontent.com/2008881/142306526-a20b784c-4875-4874-8d3a-87156788eb4e.png)

Can you attach a screenshot showing the inspector so we have more detail?

**Comment by bkjohnson** [comment]
I also noticed that the headers in our screenshots were different - yours had "Sign in to VA.gov" and mine just had "Sign in". I did some digging and found that it looks like we're still using the same modal, but it [has had some changes over the last week or two](https://github.com/department-of-veterans-affairs/vets-website/commit/faca5657885cec7a3dab013144abce3fefdfe854#).

It looks like in that time, an [a11y update to the modal](https://github.com/department-of-veterans-affairs/component-library/commit/594428875a479f1bd1595354db44c999909993b5#) made its way into `vets-website`.

- [`component-library` release](https://github.com/department-of-veterans-affairs/component-library/releases/tag/v3.6.0)
- [`vets-website` update](https://github.com/department-of-veterans-affairs/vets-website/pull/19321)

This no longer seems like its an issue - I'll leave this open for you to confirm.

**Comment by AGarcia-Clarity** [comment]
@bkjohnson Yes, I can confirm the `component-library` release updated the modal (in this PR https://github.com/department-of-veterans-affairs/component-library/pull/198) and it is no longer an issue.  Thank you and Erik for your awesomeness 🙌 .
